src/tests/ftest/cart/corpc/cart_corpc_five_node.yaml

```diff
@@ -4,7 +4,7 @@
 defaultENV:
   #!filter-only : /run/envs_CRT_CTX_SHARE_ADDR/no_sep
   #!filter-only : /run/tests/corpc_prefwd
-  D_LOG_MASK: "DEBUG,MEM=ERR"
+  D_LOG_MASK: "ERR,RPC=DEBUG,HG=DEBUG"
```

Comment:
Possibly WARN rather than ERR

Reply:
Good point, warns wont produce that much extra log and would be useful to see in case of problems. Changed